#!/usr/bin/env python
PACKAGE = "path_executer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_vel_x", double_t, 0, "robot's max forward velocity", 0.8 , 0.0, 3.0)   # 0.8
gen.add("max_vel_phi", double_t, 0, "robot's max rotational velocity", 1.2, 0.0, 2.5) # 1.2
gen.add("k_rho", double_t, 0, "controller gain for distance", 2.5, 0.0, 20.0)
gen.add("k_alpha", double_t, 0, "controller gain for angle to face next waypoint", 2.0, 0.0, 20.0)
gen.add("k_beta", double_t, 0, "controller gain for angle diff to next waypoint", 2.0, 0.0, 20.0)

exit(gen.generate(PACKAGE, "path_executer", "PathExecuter"))
